src/Symfony/Component/HttpKernel/ControllerMetadata/ArgumentMetadataFactory.php

```diff
@@ -65,6 +65,23 @@ private function hasDefaultValue(\ReflectionParameter $parameter)
     }
 
     /**
+     * Returns if the argument is allowed to be null but is still mandatory.
+     *
+     * @param \ReflectionParameter $parameter
+     *
+     * @return bool
+     */
+    private function isNullable(\ReflectionParameter $parameter)
+    {
+        if (PHP_VERSION_ID >= 70000) {
```

Comment:
https://github.com/iltar/symfony/blob/d4466c0d58c41d9da4fd118f1e8973f375d6bc97/src/Symfony/Component/HttpKernel/ControllerMetadata/ArgumentMetadataFactory.php#L52
https://github.com/iltar/symfony/blob/d4466c0d58c41d9da4fd118f1e8973f375d6bc97/src/Symfony/Component/HttpKernel/ControllerMetadata/ArgumentMetadataFactory.php#L105

Already doing here as well. Tbh I rather don't make hacks for hhvm.